scripts/exregional_run_analysis.sh

```diff
@@ -516,7 +516,7 @@ done
 ifsatbufr=.false.
 
 # Set some parameters for use by the GSI executable and to build the namelist
-grid_ratio=1
+grid_ratio_fv3=2
```

Comment:
@hu5970 this line actually should be removed. 
grid_ratio_fv3 is set to 2.0 by default in config_defaults.sh
And if we want to change it, it is better to change it in config.sh.

Similarly, we need to move the next line "cloudanalysistype=0" into config.sh
I will make a PR on this.